apps/condo/migrations/20240307165344-0372_auto_20240307_1153.js

```diff
@@ -0,0 +1,57 @@
+// auto generated by kmigrator
+// KMIGRATOR:0372_auto_20240307_1153:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMC41IG9uIDIwMjQtMDMtMDcgMTE6NTMKCmZyb20gZGphbmdvLmRiIGltcG9ydCBtaWdyYXRpb25zCgoKY2xhc3MgTWlncmF0aW9uKG1pZ3JhdGlvbnMuTWlncmF0aW9uKToKCiAgICBkZXBlbmRlbmNpZXMgPSBbCiAgICAgICAgKCdfZGphbmdvX3NjaGVtYScsICcwMzcxX2FsdGVyX25vdGlmaWNhdGlvbmFub255bW91c3NldHRpbmdfbWVzc2FnZXR5cGVfYW5kX21vcmUnKSwKICAgIF0KCiAgICBvcGVyYXRpb25zID0gWwogICAgXQo=
+
+exports.up = async (knex) => {
+    await knex.raw(`
+        BEGIN;
+        
+        SET statement_timeout = '1500s';
+        
+        --
+        -- [CUSTOM] Set unitType to NULL in Tickets and Contacts which have unitName null and not null unitType 
+        --
+        UPDATE "Ticket"
```

Comment:
Why `Ticket` and `Contact` only? Don't we need to migrate the rest models to make them synced with new code rules?

Reply:
I checked the database for bad data - they were found only in these schemas.
However, I can add migrations for other schemes :)